### ISSUE TITLE
comma separated CLI flags not whitespace-trimmed

### ISSUE THREAD

**Issue by deleteman** [ethereum/go-ethereum#2441]
#### System information

Geth version: `geth version`
Geth
Version: 1.4.0-unstable
Protocol Versions: [63 62 61]
Network Id: 1
Go Version: go1.5.1
OS: linux
GOPATH=
GOROOT=/usr/lib/go

OS & Version: Linux, CentOS 6

#### Expected behaviour
A success message.

#### Actual behaviour
When executing the following code, I get the error response from the title:

 submitted = contractInstance.SubmitHours.sendTransaction(
                        weekId,
                        equity,
                        weekHours,
                        {
                            from: memberAddress,
                            value: web3.toWei(0.0001, "ether"),
                            gas: 1000000
                        }
                    );

---------------------------------------------
Basically, I'm trying to understand what could cause the error message from the title of this issue, the RPC interface is loaded, infact, here is how I'm running geth:

geth --rpc --rpcapi "personal, web3, eth, net, db"

Thanks

**Comment by bas-vk** [comment]
Could you try to deploy and use the contract from the tutorial and see of that works (you need to have solc installed)? If this works it would be nice if you can post the code that you use to deploy and instantiate the web3 contract object.

```
var source = 'contract Multiply7 { event Print(uint); function multiply(uint input) returns (uint) { Print(input * 7); return input * 7; } }';
var compiled = web3.eth.compile.solidity(source);
var code = compiled.Multiply7.code;
var abi = compiled.Multiply7.info.abiDefinition;

web3.eth.contract(abi).new({from: eth.coinbase, data: code}, function (err, contract) {
   if (!err && contract.address)
      console.log("contract address:", contract.address);
   }
);

var Multiply7 = web3.eth.contract(compiled.Multiply7.info.abiDefinition);
var multi = Multiply7.at("<replace with contract address>")
multi.multiply.sendTransaction(6, {from: eth.coinbase})
```

**Comment by karalabe** [comment]
@deleteman Please delete the whitespace from `geth --rpc --rpcapi "personal, web3, eth, net, db"` to `geth --rpc --rpcapi "personal,web3,eth,net,db"` (you can even drop the quotes to `geth --rpc --rpcapi personal,web3,eth,net,db`). Does this solve the issue?

**Comment by deleteman** [comment]
@karalabe yes! That solved the issue!!! Thanks, this was driving me crazy!!

**Comment by karalabe** [comment]
Awesome :) I think we can keep this issue open as a bug report/reminder to cut off whitespace from split CLI flags.
